secure-embedded-cache/keycloak-setup/install-demo-realm.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+kcadm.sh config credentials --server http://localhost:8180/auth --realm master --user johndoe --client admin-cli
+
+#Create realm demo
+kcadm.sh create realms -s realm=demo -s enabled=true
+
+#Create client
+kcadm.sh create clients -r demo -f ${SCRIPT_DIR}/client-quickstart.json
+
+#Create roles
+kcadm.sh create roles -r demo -f ${SCRIPT_DIR}/role-admin.json
+kcadm.sh create roles -r demo -f ${SCRIPT_DIR}/role-reader.json
+
+#Create users
+kcadm.sh create users -r demo -f ${SCRIPT_DIR}/user-admin.json
+kcadm.sh create users -r demo -f ${SCRIPT_DIR}/user-reader.json
+
+#Assign roles
+kcadm.sh add-roles -r demo --uusername admin --rolename admin
```

Comment:
Actually, you can embed users into an exported realm. Here's an example: https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/test/resources/testrealm.json#L22

Reply:
Yeah. That's true. But I wanted to approach this from the end user who needs to create roles and users. Exporting a realm and then adding roles/users to it seems like we're not starting from scratch. Also, when I export the realm it is so long and there are so many elements that 
a) it's confusing for the end user (IMO)
b) personally I don't know which elements I can remove to make it concise and still work

Reply:
Fair point. I'll leave it up to you.

However, if you'd like to create the smallest json file for import, this should do the trick: https://github.com/stianst/keycloak-demo/blob/master/demo-realm.json

Reply:
Thanks for the tip.